decidim-core/lib/tasks/decidim_reminders_tasks.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+namespace :decidim do
+  namespace :reminders do
```

Comment:
As it's a new cron job, we should add it to the CHANGELOG so implementers can know about this new feature when upgrading